src/cmd_sym.ml

```diff
@@ -197,50 +197,49 @@ let cmd profiling debug unsafe optimize workers no_stop_at_failure no_values
   let result = List.fold_left (run_file ~unsafe ~optimize) pc files in
   let thread : Thread.t = Thread.create () in
   let results = Choice.run ~workers result thread in
-  let* failing =
-    list_fold_left
-      (fun failing (result, thread) ->
-        let pc = Thread.pc thread in
-        let symbols = thread.symbol_set in
-        let model = get_model ~symbols solver pc in
-        let* result, err =
-          match result with
-          | Symbolic_choice.Multicore.EVal (Ok ()) -> Ok (failing, false)
-          | EAssert assertion ->
-            Ok (`EAssert (assertion, model) :: failing, true)
-          | ETrap tr -> Ok (`ETrap (tr, model) :: failing, true)
-          | EVal (Error e) -> Error e
-        in
-        if not no_values then
-          let testcase =
-            List.sort compare (Encoding.Model.get_bindings model)
-            |> List.map snd
-          in
-          let+ () = write_testcase ~dir:workspace ~err testcase in
-          result
-        else Ok result )
-      [] (List.of_seq results)
-  in
   let print_bug = function
     | `ETrap (tr, model) ->
       Format.pp_std "Trap: %s@\n" (Trap.to_string tr);
       Format.pp_std "Model:@\n  @[<v>%a@]@\n"
         (Encoding.Model.pp ~no_values)
-        model
+        model;
+      Format.pp_flush Stdlib.Format.std_formatter ()
```

Comment:
This should not be needed.

Reply:
Seq is lazy. When I add a `flush`, the display is progressive. Is there a better way to do this?

Reply:
I don't understand what this has to do with `Seq` being lazy ? The display should be strict even without the flush here.

Reply:
I don't know :( Without the `flush`, the display is not progressive and appears all at once at the end.
If there is a better way to do this, no problem ...

Reply:
It's also my experience that `@\n` is buffered and the output only appears at the end. I tend to use `@.` to insert newlines when I'm terminating a formatter, it supposedly also does a `flush`. 

To this end, we could terminate the model print with `@.` (i.e., `Format.pp_std "Model:@\n  @[<v>%a@]@."`) 

It has the same intended format of being a progressive print, but its exactly the same to what is being done here.

Reply:
Thanks @filipeom !
Works great with `@.`